src/model.js

```diff
@@ -24,29 +32,44 @@ function Model () {}
  * @param {function} callback
  */
 Model.prototype.getData = function (req, callback) {
-  let filename = `${req.params.id}.geojson`
-  fs.readFile(`${dataDir}/${filename}`, (err, dataBuffer) => {
+  const filename = `${req.params.id}.geojson`;
+  const filePath = `${this.dataDirPath}/${filename}`
```

Comment:
why not combine these into a single string template?

Reply:
`filename` is used as option for `geojson.metadata.name`.